src/umpire/Allocator.inl

```diff
@@ -11,45 +11,21 @@
 
 #include "umpire/Allocator.hpp"
 
-#include "umpire/util/Macros.hpp"
-#include "umpire/Replay.hpp"
-
 namespace umpire {
 
-inline void*
```

Comment:
Does inline have to be removed? We need the symbol `umpire_ver_2_found` to show up in user code for our mismatch detection to work.

Reply:
I'd be surprised if it did (it could have been Al trying to circumvent the mismatch detection before he figured out that he needed to change our windows specific config file)